device/ospfv3/router.yaml

```diff
@@ -0,0 +1,103 @@
+components:
+  schemas:
+    Device.Ospfv3Router:
+      description: >-
+        A container of properties for an OSPFv3 router and its interfaces & Route Ranges.
+      x-status:
+        status: under_review
+        information: OSPFv3 is currently under review for pending exploration on use cases.
+      type: object
+      required: [name, router_id, interfaces]
+      properties:
```

Comment:
Probably we need to start with array of instances and then have router within it. Otherwise we will not be able to support OSPFv3 Multi instance in any reasonable way.

Reply:
updated in latest version.